src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt

```diff
@@ -198,19 +207,21 @@ object OverlayButtonsPatch : BaseResourcePatch(
                         )
 
                         // Adjust TimeBar and Chapter bottom padding
-                        val timBarItem = mutableMapOf(
-                            "@id/time_bar_chapter_title" to "16.0dip",
-                            "@id/timestamps_container" to "14.0dip"
-                        )
+						if (WiderButtonsSpace == false) {
```

Comment:
According to the code when the PR was first opened, bottom margin was set only when `WiderButtonsSpace` was FALSE